src/monorepo-npm-publisher/src/index.js

```diff
@@ -54,7 +54,7 @@ export default async function publish(options: Options) {
         npmAuthToken: options.npmAuthToken,
       });
 
-      if (!semver.gt(packageJSONFile.version, data.latest)) {
+      if (!semver.gt(packageJSONFile.version, data.latest[0])) {
```

Comment:
There are some incompatibilities. I also don't know how to install our packages from the registry yet.

Reply:
you mean this? https://help.github.com/en/github/managing-packages-with-github-package-registry/configuring-npm-for-use-with-github-package-registry#installing-a-package

also, isn't it complication for potential adopters - having to set this just to install it?

Reply:
Evaluating - probably yes, so far it's quite annoying.